packages/dx-react-grid/docs/guides/fundamentals.md

```diff
@@ -18,7 +18,11 @@ The `TableHeaderRow` plugin should follow the `Table` plugin. See the [Plugin Or
 
 The Grid's visualization plugins provide a rich API to customize grid elements' appearance. Examples of the most popular customization tasks are described below.
 
-The `Table` plugin allows you to customize table cell appearance using the `cellComponent` property. For instance, you can implement conditional cell formatting as demonstrated in the following example:
+The `Table` plugin allows you to customize table, table head and table body appearance using `tableComponent`, `tableHead`, `tableBody` and `containerComponent` [properties](../reference/table.md/#properties) of Table. The following sample demonstrated how to use `tableComponent` and make a 'striped' table.
```

Comment:
tableHeadComponent? tableBodyComponent? Missed semicolon at the end of the paragraph